google/cloud/storage/benchmarks/aggregate_upload_throughput_benchmark.cc

```diff
@@ -298,6 +285,22 @@ int main(int argc, char* argv[]) {
 
 namespace {
 
+std::string ExtractPeer(
+    std::multimap<std::string, std::string> const& headers) {
+  auto p = headers.find(":grpc-context-peer");
+  if (p == headers.end()) {
+    p = headers.find(":curl-peer");
+  }
+  return p == headers.end() ? std::string{"[peer-unknown]"} : p->second;
+}
+
+std::string ExtractUploadId(std::string v) {
+  auto constexpr kRestField = "upload_id=";
+  auto const pos = v.find(kRestField);
```

Comment:
Q.  Should this be more like a `StartsWith`/`ConsumePrefix`?

Reply:
There is a lot of (unfortunately variable) stuff before `upload_id=` that I also need to remove, I do not think `ConsumePrefix()` works in this case, and this seems simple enough.

Reply:
Ack.  I was confused that if there is stuff before the `upload_id=`, why there might not be stuff after the value too.  But I'm only assuming so.